src/SFA.DAS.EmployerIncentives.Api.UnitTests/Controllers/PassThroughControllerTests.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System.Threading.Tasks;
+using AutoFixture;
+using FluentAssertions;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.EmployerIncentives.Api.Controllers;
+using SFA.DAS.EmployerIncentives.Infrastructure.Api;
+using SFA.DAS.EmployerIncentives.Interfaces;
+using SFA.DAS.EmployerIncentives.Models.PassThrough;
+
+namespace SFA.DAS.EmployerIncentives.Api.UnitTests.Controllers
+{
+    [TestFixture]
+    [Parallelizable(ParallelScope.All)]
+    public class PassThroughControllerTests
+    {
+        [Test]
+        public async Task When_AddLegalEntity_is_requested_Then_EmployerIncentivesPassThroughService_is_called()
+        {
+            var f = new TestsFixture().SetupEmployerIncentivesServiceForAddLegalEntityCall();
```

Comment:
No fixtures - these are bloated, complex and confusing. Use the SFA.DAS.Testing package for any auto mocking

Reply:
The only thing I would say, is they can keep the tests down to a few lines and it 'sometimes' reads much better